lib/logger/lib/logger/translator.ex

```diff
@@ -534,6 +546,9 @@ defmodule Logger.Translator do
     end
   end
 
+  defp registered_name(name) when is_atom(name), do: [registered_name: name]
+  defp registered_name(_name), do: []
```

Comment:
What about via tuples here? Are we ignoring them?

Reply:
@michalmuskala yes, as the name can be any term and it is not guaranteed we can log those. So I am sticking with the otp reports conventions here (either an atom or nothing).